pages/signup/_user.vue

```diff
@@ -2,7 +2,7 @@
   <div class="page-outer">
     <div class="flex-box">
       <div class="signup container-default vertical-flex">
-        <div class="top-heading">
+        <div class="signup__title top-heading">
```

Comment:
I encounter a bug causing the sign-up button to not become active, please take a look.